enhancements/installer/azurestack.md

```diff
@@ -0,0 +1,160 @@
+---
+title: azurestack
+authors:
+  - "@patrickdillon"
+reviewers:
+  - "@staebler"
+  - "@jhixson74"
+approvers:
+  - "@staebler"
+  - "@crawford"
+creation-date: 2021-03-10
+last-updated: 2021-03-10
+status: implementable
+see-also:
+  - "/enhancements/installer/azure-support-known-cloud-environments.md"
+---
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement covers adding support to install OpenShift clusters to Azure Stack Hub (ASH).
+Azure Stack Hub looks and feels like standard Azure, but differs significantly in terms of
+implemenetation and technical details. Azure Stack Hub shares characteristics from [Azure cloud
+environments](azure-support-known-cloud-environments.md) and on-prem platforms. This enhancement
+covers details for the Installer to create infrastructure and run an OpenShift cluster on ASH.
+
+## Motivation
+
+Currently, Azure Stack Hub is not supported but is a significant on-prem platform.
+
+### Goals
+
+1. Installer creates infrastructure and installs OpenShift on Azure Stack Hub.
+2. All standard support that entails, such as destroy.
+
+### Non-Goals
+
+1. ASH will not "inherit" established Azure
+features such as using existing networks or resource groups.
+
+## Proposal
+
+
+### User Stories
+
+#### Story 1
+As a cluster admin, I want to be able to use `openshift-install` to create and destroy a cluster on
+Azure Stack Hub.
+
+#### Story 2
+As an OpenShift developer, I want Azure Stack Hub to be managed and released like other platforms and covered by CI tests.
+
+### Implementation Details/Notes/Constraints [optional]
+
+Primary ways in which Azure Stack differs from Azure:
```

Comment:
I would also list the need to specify the API endpoints as a way in which ASH differs. That difference is significant.